examples/app-basic/tests/appConfig.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { test, expect } from './fixtures';
+
+test('configuring app', async ({ appConfigPage, page }) => {
+  await appConfigPage.goto();
```

Comment:
Can we move the `goto` method call to the fixture (we do that for all other page fixtures)?